cargo-nextest/src/dispatch.rs

```diff
@@ -762,6 +765,57 @@ impl TestBuildFilter {
     }
 }
 
+#[derive(Debug, Args)]
+#[command(next_help_heading = "Filter options")]
+struct ArchiveBuildFilter {
+    /// Test filterset (see {n}<https://nexte.st/docs/filtersets>).
+    #[arg(
+        long,
+        alias = "filter-expr",
+        short = 'E',
+        value_name = "EXPR",
+        action(ArgAction::Append)
+    )]
+    filterset: Vec<String>,
+
+    /// Ignore the default filter configured in the profile.
+    ///
+    /// By default, all filtersets are intersected with the default filter configured in the
+    /// profile. This flag disables that behavior.
+    ///
+    /// This flag doesn't change the definition of the `default()` filterset.
+    #[arg(long)]
+    ignore_default_filter: bool,
+
+    /// Test name filters.
+    #[arg(help_heading = None, name = "FILTERS")]
+    pre_double_dash_filters: Vec<String>,
+
+    /// Test name filters and emulated test binary arguments.
+    ///
+    /// Supported arguments:{n}
+    /// - --ignored:         Only run ignored tests{n}
+    /// - --include-ignored: Run both ignored and non-ignored tests{n}
+    /// - --skip PATTERN:    Skip tests that match the pattern{n}
+    /// - --exact:           Run tests that exactly match patterns after `--`
+    #[arg(help_heading = None, value_name = "FILTERS_AND_ARGS", last = true)]
+    filters: Vec<String>,
```

Comment:
Hmm, we can't support these either I'm pretty sure.

Reply:
Removed